src/dnx.windows/dnx.exe.manifest

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1" xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
+    <!-- Compatibility manifests so we get the right behavior out of APIs -->
+    <!-- See: https://msdn.microsoft.com/en-us/library/windows/desktop/dn481241(v=vs.85).aspx -->
+    <compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1"> 
+        <application> 
+            <!-- Windows 10 --> 
```

Comment:
This was the simplest way I found back then and I don't know if there is a different way. On the other hand Windows 10 is onecore too. The code between onecore and win32 is the same. The only difference is what we link against.